src/styles/main.scss

```diff
@@ -0,0 +1,103 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.product {
+  @include flex-column;
+
+  width: $product-full-width;
+  background-color: $background;
+  align-items: center;
+  text-align: center;
+  padding-block: 34px 0;
+  &-image {
+    width: $product-full-width * 0.8;
+  }
+
+  &-info {
+    @include flex-column;
+
+    width: $product-full-width * 0.83;
+    text-align: justify;
+  }
+
+  &-title {
+    font-size: $middle-size-font;
+    line-height: 18px;
+    font-family: Roboto, sans-serif;
+    font-weight: 500;
+    color: #060b35;
+    padding-block: 40px 4px;
+  }
+
+  &-code {
+    font-size: $small-size-font;
+    line-height: 14px;
+    color: #616070;
+    font-family: Roboto, sans-serif;
+  }
+
+  &-star {
+    padding-block: 16px 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: $product-full-width * 0.8;
+  }
+
+  &-reviews {
+    font-family: Roboto, sans-serif;
+    font-size: $small-size-font;
+    line-height: 16px;
+    color: #060b35;
+    font-weight: 400;
+  }
+
+  &-price {
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      font-family: Roboto, sans-serif;
```

Comment:
You don't need to doublicate font-family property. Just use it in for parent (element with class card)

Reply:
The same is for other places where you using font-family parametr.